tests/unit/s2n_tls13_pq_handshake_test.c

```diff
@@ -590,7 +554,7 @@ int main()
          * ECC on hello retry.
          */
         {
-                .client_policy = &security_policy_pq_tls_1_1_2021_05_21,
+                .client_policy = &security_policy_pq_tls_1_0_2021_05_24,
```

Comment:
For your tests: how did you verify that you have the right replacement policy, and aren't changing the behavior of the test?